lib/matplotlib/patches.py

```diff
@@ -461,8 +462,7 @@ def set_capstyle(self, s):
         ----------
         s : {'butt', 'round', 'projecting'}
         """
-        s = mpl.rcsetup._deprecate_case_insensitive_join_cap(s)
-        cbook._check_in_list(self.validCap, capstyle=s)
```

Comment:
Not clear how (or if worth it) to deprecate `Line2D.validCap` and `Line2D.validJoin` (lines 34-35).

Reply:
let's not bother...